packages/clerk-js/src/ui/elements/Modal.tsx

```diff
@@ -50,15 +50,13 @@ export const Modal = withFloatingTree((props: ModalProps) => {
     >
       <ModalContext.Provider value={modalCtx}>
         <Flex
-          aria-hidden
```

Comment:
I don't think we want this, as it removes the entire modal from the accessibility tree.

Reply:
Definitely not 😅